packages/utils/src/s3/S3Client.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Tupaia MediTrak
+ * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
+ */
+
+import fs from 'fs';
+import path from 'path';
+import { getS3UploadFilePath, getS3ImageFilePath, S3_BUCKET_NAME } from './constants';
+import { getUniqueFileName } from './getUniqueFileName';
+
+export class S3Client {
```

Comment:
I had been tempted to put this all in its own package? But I think utils is better for now. I think in Tupaia our general plan is to keep packages mainly for referring to nodes in our data-flow, rather than for random libraries...